src/containers/PsychoAssess/sections/Questions.jsx

```diff
@@ -1,16 +1,119 @@
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>{showResults(calculateScore())}</Modal.Body>
+        <Modal.Footer>
+          <Link to="/contact">
+            <Button className="testContactButton" variant="primary">
+              {t("assessment.contactButton")}
+            </Button>
+          </Link>
+          <Button className="testDoneButton" onClick={props.onHide}>
+            {t("assessment.closeButton")}
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  };
+
+  const showResults = (score) => {
+    if (score !== -15) {
+      if (score >= 52) {
+        return (
+          <>
+            <span className="resultCategory">
+              {t("assessment.verySevere.2")}
+            </span>
+            <h3>{t("assessment.verySevere.0")}</h3>
+            <h6>{t("assessment.submitSuccess")}</h6>
+            <h4>{t("assessment.verySevere.1")}</h4>
+          </>
+        );
+      } else if (score >= 25) {
+        return (
+          <>
+            <span className="resultCategory">{t("assessment.severe.2")}</span>
+            <h5>{t("assessment.severe.0")}</h5>
+            <h6>{t("assessment.submitSuccess")}</h6>
+            <p>{t("assessment.severe.1")}</p>
+            <Link to="/contact">
+              <Button className="testSubmitButton" variant="primary">
+                {t("assessment.contactButton")}
+              </Button>
+            </Link>
+          </>
+        );
+      } else if (score >= 18) {
+        return (
+          <>
+            <span className="resultCategory">{t("assessment.moderate.2")}</span>
+            <h5>{t("assessment.moderate.0")}</h5>
+            <h6>{t("assessment.submitSuccess")}</h6>
+            <p>{t("assessment.moderate.1")}</p>
+          </>
+        );
+      } else if (score >= 13) {
+        return (
+          <>
+            <span className="resultCategory">{t("assessment.mild.2")}</span>
+            <h5>{t("assessment.mild.0")}</h5>
+            <h6>{t("assessment.submitSuccess")}</h6>
+            <p>{t("assessment.mild.1")}</p>
+          </>
+        );
+      } else if (score >= 7) {
+        return (
+          <>
+            <span className="resultCategory">{t("assessment.light.2")}</span>
+            <h5>{t("assessment.light.0")}</h5>
+            <h6>{t("assessment.submitSuccess")}</h6>
+            <p>{t("assessment.light.1")}</p>
+          </>
+        );
+      }
+    }
+    return <h5>{t("assessment.answerMissing")}</h5>;
+  };
+
+  let userScore = 0;
+  const calculateScore = () => {
+    const radioButtons = document.getElementsByClassName("testChoice");
+    let checkedRadioNumber = 0;
+    for (let i = 0; i < radioButtons.length; i++) {
+      if (radioButtons.item(i).checked) {
+        userScore += parseInt(radioButtons.item(i).value);
+        checkedRadioNumber++;
+      }
+    }
+    if (checkedRadioNumber === QUESTION_QUANTITY) {
+      return userScore;
+    }
+    return -15;
```

Comment:
please no magic numbers, -15 is in here hard-coded twice. i have no idea what -15 means

Reply:
done, please take a look when available.

Reply:
done